doc/eslint.config_partial.mjs

```diff
@@ -23,6 +23,8 @@ export default [
         },
       ],
       'no-undef': 'off',
+      '@typescript-eslint/no-unused-expressions': 'off',
+      '@typescript-eslint/no-unused-vars': 'off',
```

Comment:
Why do we disable these?

Reply:
These are the TypeScript equivalent of the two rules below.